### PR TITLE
Mobile menu/feature

### PR DESCRIPTION
**Pull request by denscholar**
### Mobile Menu features
- When the user clicks (or taps) the hamburger button, the mobile menu appears.
- When the user clicks (or taps) the close (X) button, the mobile menu disappears.
- When the user clicks (or taps) any of the mobile menu options, the mobile menu disappears.
- When the user clicks (or taps) any of the mobile menu options, a correct part of the page is displayed.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
+const menuBar = document.querySelector('.menu-bar');
```
Comment (carlos-ssh) [inline index.js]: **KINDLY WORK ON THESE CHANGES:**
_Provide the requested design:_
**_Yours:_**
Screen Shot 2021-12-13 at 22 04 33

**_Figma Design:_**
Screen Shot 2021-12-13 at 22 05 22

**_Fit your `menu` at the `body`._**

**Review by carlos-ssh — CHANGES_REQUESTED** [review]
# 🦺 🚧 🚥 CHANGES REQUIRED 🚥 🚧 🦺
#

Hi @denscholar, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Required Changes ♻️
_Check the comments under the review._ 


changesrequested
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: css/styles.css [hunk 2]
```diff
@@ -22,10 +22,64 @@ header .nav a {
...
+.menu-bar {
+  position: absolute;
+  display: flex;
+  flex-direction: column;
+  top: 0%;
+  right: 0%;
+  width: 100%;
+  height: 820px;
+  z-index: 10;
+  background-color: white;
```
Comment (rotimiazeez) [inline css/styles.css]: screenshot (9)

- [x] I'd suggest you use `height: 100%` and the `position: fixed` so the menu fills the screen and it does not scroll with the page🤔

**Review by rotimiazeez — CHANGES_REQUESTED** [review]
# Hi TEAM👋, 

### Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

> Kindly make the suggested changes and you are good to go🚀
 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: css/styles.css (hunk 2 above)
Comment (denscholar) [reply css/styles.css]: Ok. Thank you. Fixed.

**Review by rotimiazeez — APPROVED** [review]
# Hi TEAM🔥, 

### Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 🚀
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**